usr.bin/man/man.sh

```diff
@@ -739,10 +739,9 @@ man_setup_locale() {
 # Display usage for the man utility.
 man_usage() {
 	echo 'Usage:'
-	echo ' man [-adho] [-t | -w] [-K regexp] [-M manpath] [-P pager] [-S mansect]'
+	echo ' man [-adho] [-t | -w] [-M manpath] [-P pager] [-S mansect]'
 	echo '     [-m arch[:machine]] [-p [eprtv]] [mansect] page [...]'
-	echo ' man -f page [...] -- Emulates whatis(1)'
-	echo ' man -k page [...] -- Emulates apropos(1)'
+	echo ' man -K | -f | -k expression [...] -- Search manual pages'
```

Comment:
Is there any chance 'expression' is confusing and should be 'regexp' ? either are better than 'page', that's for sure.

Reply:
I agree that regexp is less ambiguous for people new to unix, but the term expression is elegant and used in all the rest of the manual manuals like apropos. Also, `man -k expression` does show `re_format(7) - POSIX 1003.2 regular expressions`.